decidim-proposals/spec/system/admin/valuator_manages_proposals_spec.rb

```diff
@@ -113,5 +113,20 @@
       end
       expect(page).to have_content("successfully")
     end
+
+    it "have assign button" do
+      expect(page).to have_content("Assign")
+    end
```

Comment:
```suggestion
    it "have assign button" do
      expect(page).not_to have_content("Assign")
    end
```

Evaluator should not have access to add other evaluators. This tests is handling the case of the user logged is a valuator. So he/she shouldn't be able to assign other valuators